src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 7) {
+    total = (days * 40) - 50;
+  } else if (days >= 3 && days <= 6) {
+    total = (days * 40) - 20;
+  } else {
+    total = days * 40;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return (days * 40) - 50;
  } 
  
  if (days >= 3 && days <= 6) {
    return (days * 40) - 20;
  } 
 
 return days * 40;
  
```

Reply:
check the previous comment

declare variables with descriptive names and assign these values
<img width="298" alt="image" src="https://user-images.githubusercontent.com/102411130/215695472-a13a65d1-a022-454b-aa6e-95ab23184077.png">